packages/auth/src/auth-options.ts

```diff
@@ -1,39 +1,57 @@
 import type { ServiceLayer } from "@dotkomonline/core"
-import type { DefaultSession, DefaultUser, NextAuthOptions, User } from "next-auth"
+import type { NextAuthOptions } from "next-auth"
 import Auth0Provider from "next-auth/providers/auth0"
 
-interface Auth0IdTokenClaims {
-  given_name: string
-  family_name: string
-  nickname: string
-  name: string
-  picture: string
-  gender: string
-  updated_at: string
-  email: string
-  email_verified: boolean
+// https://www.iana.org/assignments/jwt/jwt.xhtml
```

Comment:
Not that important but I prefer we use the ID Token spec link https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims

Reply:
👍🏻